packages/shared-ui/src/i18n/en.json

```diff
@@ -73,9 +73,11 @@
     },
     "email-verify": {
       "email-otp": {
-        "header": "Enter code to create account",
-        "body_title": "One-time passcode",
-        "body_description": "Enter the passcode sent to your email address",
+        "common": {
```

Comment:
Technically, this is a breaking change => I think we should keep the old keys as they are and add the new translations for "postLoginVerification" as you did it. Even if your solution is better I would make the changes backwards-compatible if possible.
So I would do this:
 "email-otp": {
        "header": "Enter code to log in",
        "body_title": "One-time passcode",
        "body_description": "Enter the passcode sent to your email address",
        "header": "Enter code to log in",
        "body_title": "One-time passcode",
        "body_description": "Enter the passcode sent to your email address"
        "postLoginVerification": {
          "header": "Your email is not verified yet",
          "body_title": "Verify your email address",
          "body_description": "Please verify your email by entering the one-time passcode we just sent to your email."
        },